src/Audio/CueData.cs

```diff
@@ -663,36 +682,98 @@ public XACTClip(BinaryReader reader, double clipVolume, byte filterType)
 						weights
 					);
 				}
-				else if (eventType == 7) // SetPitchEvent
```

Comment:
Note that I converted these eventType constants to hex in the hopes that I would see  a bit pattern relating the non-repeating with the repeating codes, but there doesn't seem to be much of a relationship, just arbitrary event codes.  I think rather than change them back, I would just move forward to an enum to represent all known codes.